params/protocol_params.go

```diff
@@ -43,6 +47,15 @@ var (
 	MinGasLimit     = big.NewInt(5000)                  // Minimum the gas limit may ever be.
 	GenesisGasLimit = big.NewInt(4712388)               // Gas limit of the Genesis block.
 	TargetGasLimit  = new(big.Int).Set(GenesisGasLimit) // The artificial target
+	HardGasLimit    *big.Int                            // Hard gas limit never to allow going over
```

Comment:
OK well in that case maybe put in new file in package `params` ? Just want to avoid having miner strategy params in a file called "protocol_params" and right next to consensus protocol params. Came to think of it, we should actually move `TargetGasLimit` too since it's also miner strat.